osu.Game/Screens/Menu/OsuLogo.cs

```diff
@@ -391,6 +393,8 @@ protected override bool OnClick(ClickEvent e)
 
         protected override bool OnHover(HoverEvent e)
         {
+            if (Action == null) return false;
```

Comment:
Why is this a `false` return versus the `true` below? Seems weird.

Reply:
Yeah I'm not sure on whether the logo should block components behind it or not, but since the current result matches how it behaved previously, I overlooked it as to not be nit-picky.

Reply:
I think I'd rather it's consistent here.

Reply:
I believe this also includes other events, since they're inconsistent right now. No?